environment.yml

```diff
@@ -40,3 +40,4 @@ dependencies:
   - boost-cpp
   - libgomp
   - tqdm
+  - joblib <=1.1
```

Comment:
Note that this is temporary and dependent on hdbscan

Reply:
Also I've got a PR #236 which didn't manage to resolve the environment in 6 hrs so ran out of time, so currently I'm not particularly confident we can sort this until that's fixed